### ISSUE TITLE
RNN ERROR: AssertionError: vec(value(x)) isa WTYPE

### ISSUE THREAD

**Issue by hack1nt0** [denizyuret/Knet.jl#482]
```
rnn = RNN(1, 2)
in = ones(Float32, 1)
rnn(KnetArray(in)) # works ok

rnn2 = cpucopy(rnn)
rnn2(in) # throw error as show in title
```
I manually check that the source code in rnn.jl 
```
@assert vec(value(x)) isa WTYPE
```
is passed, is this a issue or a mistake usage?

**Comment by hack1nt0** [comment]
Knet : v1.2.4
Julia : v1.1.1

**Comment by denizyuret** [comment]
Thank you. This has been fixed in the latest commit, and the fix will be in 1.2.5.

**Comment by hack1nt0** [comment]
Thx for this amazing product.
